test/appservice/AppserviceTest.ts

```diff
@@ -2125,7 +2162,284 @@ describe('Appservice', () => {
             appservice.stop();
         }
     });
+    describe("thirdparty.protocol", () => {
```

Comment:
the bot-sdk doesn't run tests structured like this. Instead, the `it()` function is used on the top level Appservice test. It also uses descriptive words for what it is doing.

See `should return room not found when a promise to not create a room is seen` as an example.